src/network/address.rs

```diff
@@ -82,10 +82,8 @@ impl Encodable for Address {
     fn consensus_encode<S: io::Write>(&self, mut s: S) -> Result<usize, io::Error> {
         let len = self.services.consensus_encode(&mut s)?
             + addr_to_be(self.address).consensus_encode(&mut s)?
-
             // consensus_encode always encodes in LE, and we want to encode in BE.
-            //TODO `len += io::Write::write(&mut e, &self.port.to_be_bytes())?;` when MSRV >= 1.32
-            + self.port.swap_bytes().consensus_encode(s)?;
+            + self.port.to_be_bytes().consensus_encode(s)?;
```

Comment:
This looks incorrect because `swap_bytes` swaps the bytes regardless of the platform while `to_be_bytes` should do nothing on big endian platforms. I suspect we're missing a test case here. Remember that this change is in the PR multiple times.

Reply:
I believe `s390x-unknown-linux-gnu` is BE

Reply:
Good reviewing @Kixunil, thanks. Will add test and re-think.

Reply:
I'm pulling this patch out of the PR.

Reply:
Please see #968